o/opentelemetry-js/opentelemetry-js_ubi-8.5.sh

```diff
@@ -55,6 +55,15 @@ if ! npm run compile; then
 	exit 1
 fi
 
+# If api directory is available in the package then first need to run npm run compile in the api directory before running tests 
+# because it requires the ESM targets are generated with npm run compile in the API directory.
+  
+  if [ -d "api" ]; then
```

Comment:
@Pratikt2312 Can you verify if update script works for earlier version `v1.4.0`
If not , please check new step is specific to v1.8.0 only?

Reply:
Yes,I have verified the script again for version v1.4.0 and some other versions so its working fine.

Reply:
Good. Then you just need to add support of older version in your build_info

Reply:
Already have the support for older version in json file.